language/tools/move-cli/src/main.rs

```diff
@@ -103,9 +104,14 @@ pub enum Command {
     /// This command compiles the script first before running it.
     #[structopt(name = "run")]
     Run {
-        /// Path to script to compile and run.
+        /// Path to .mv file containing either script or module bytecodes. If the file is a module, the
+        /// `script_name` parameter must be set.
         #[structopt(name = "script")]
         script_file: String,
+        /// Name of the scrupt function inside `script_file` to call. Should only be set if `script_file`
+        /// points to a module.
+        #[structopt(name = "name")]
+        script_name: Option<String>,
```

Comment:
I feel like this is a bit weird as files can have multiple scripts in them, so it might be helpful for disambiguating between them.

Reply:
Are you suggesting that we repurpose this to pick which function to call in a script source file with multiple scripts? I think that would be reasonable to add in a follow-up.